interface/patient_tracker/patient_tracker.php

```diff
@@ -140,6 +159,74 @@ function openNewTopWindow(newpid,newencounterid) {
 
 <body class="body_top" >
 
+<form method='post' name='theform' id='theform' action='patient_tracker.php' onsubmit='return top.restoreSession()'>
+
+    <div id="flow_board_parameters">
+        <table>
+            <tr class="text">
+                <td class='label'><?php echo xlt('Provider'); ?>:</td>
```

Comment:
If the following global is set, then don't show this option (and just set it to the provider):
``````
$GLOBALS['docs_see_entire_calendar'] !='1'